Xamarin.Forms.Platform.iOS/Renderers/NavigationRenderer.cs

```diff
@@ -619,14 +619,8 @@ void UpdateBarBackgroundColor()
 		{
 			var barBackgroundColor = NavPage.BarBackgroundColor;
 
-			if (!Forms.IsiOS13OrNewer)
-			{
-				// Set navigation bar background color
-				NavigationBar.BarTintColor = barBackgroundColor == Color.Default
-				? UINavigationBar.Appearance.BarTintColor
-				: barBackgroundColor.ToUIColor();
-			}
-			else
+#if __XCODE11__
+			if (Forms.IsiOS13OrNewer)
```

Comment:
@rmarinho can you verify all the changes to this file please? 

Basically if you are compiling on VS 2017 I'm removing all the code you checked in for this PR

https://github.com/xamarin/Xamarin.Forms/pull/9346